src/Task.gren

```diff
@@ -390,12 +391,25 @@ execute task =
     command (Execute (map (\_ -> {}) task))
 
 
+{-| This is like `execute` where we want to give a command without being told how it went,
+but follow up with a command to run once the task completes.
+
+Maybe we want to shut down a simple node program once a task is done.
+-}
+executeCmd : (a -> Cmd msg) -> Task Never a -> Cmd msg
```

Comment:
Oh, I thought we decided on `Task Never (Cmd a) -> Cmd a`.

As in not taking in a function here.

Reply:
I have not had my coffee this morning 😂